src/app/[locale]/homePage/page.tsx

```diff
@@ -12,11 +15,33 @@ const HomePage = () => {
     queryFn: fetchGames,
   });
 
-  if (isLoading) console.log('Ładowanie');
-  if (data) console.log(data);
-  if (isError) console.log('Błąd');
-
-  return <p>{t('Home Page')}</p>;
+  return (
+    <main className="w-screen py-[70px] px-[113px]">
+      <span className="text-[24px] font-[700] text-texts-hotDeals">{t('Hot deals')}</span>
+      <div className="grid gap-[60px] grid-cols-[repeat(auto-fill,minmax(200px,1fr))]">
+        {data && (
+          <>
+            {data.map((game: Game) => (
+              <div key={game._id}>
+                <GameTile game={game} />
+              </div>
+            ))}
+          </>
+        )}
+        {isLoading && (
+          <>
+            <GameTileSkeleton />
+            <GameTileSkeleton />
+            <GameTileSkeleton />
+            <GameTileSkeleton />
+            <GameTileSkeleton />
+            <GameTileSkeleton />
+            <GameTileSkeleton />
+          </>
```

Comment:
to wszystko dodajmy do komponentu HotDeals, i dodajmy jeszcze obsługę błędu z useQuery